test/module/index.js

```diff
@@ -98,44 +98,40 @@ describe('Module Tests', function() {
   it('should throw an error when the first parameter is not defined', function() {
     const lighthouse = require('../..');
     return lighthouse()
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+      .then(() => {
+        throw new Error('Should not have resolved when first arg is not a string');
+      }, err => {
+        assert.ok(err);
+      });
   });
 
   it('should throw an error when the first parameter is an empty string', function() {
     const lighthouse = require('../..');
-    return lighthouse()
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+    return lighthouse('')
+      .then(() => {
+        throw new Error('Should not have resolved when first arg is an empty string');
+      }, err => {
+        assert.ok(err);
+      });
   });
 
   it('should throw an error when the first parameter is not a string', function() {
     const lighthouse = require('../..');
     return lighthouse({})
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+      .then(() => {
+        throw new Error('Should not have resolved when first arg is not a string');
+      }, err => {
+        assert.ok(err);
+      });
   });
 
   it('should throw an error when the second parameter is not an object', function() {
     const lighthouse = require('../..');
-    return lighthouse(VALID_TEST_URL, [])
-    .then(() => {
-      assert.error(new Error('Should not have resolved when first arg is not a string'));
-    })
-    .catch(err => {
-      assert.ok(err);
-    });
+    return lighthouse(VALID_TEST_URL, 'flags')
```

Comment:
switched second argument from `[]` to `'flags'`. Lighthouse doesn't actually check what the options object is, so it'll happily add default flag values to a passed in array (as the test was originally). We could do stricter duck typing in there, but not sure of the value if it works. Used a string here as a representative for failure on primitives, frozen objects, etc